linux/LinuxProcess.c

```diff
@@ -763,6 +787,18 @@ static double adjustNaN(double num) {
    return num;
 }
 
+#ifdef HAVE_LIBELF
+static int sortElfType(elf_state_t es) {
```

Comment:
Return an architecture. Not everything is just i386 and amd64 … There's common enough Linux versions on armv7 and arm64 too.

Reply:
This is not about architecture, it is really about 64 bit vs 32 bit sized addresses.